### ISSUE TITLE
Question about Unsupervised-TU ''test'' and ''val''

### ISSUE THREAD

**Issue by jiaruHithub** [Shen-Lab/GraphCL#65]
Hi! In Unsupervised-TU, such as logistic regression classifier, there are two parts, test and val, annotated in the ten fold cross verification section. But in my opinion, these two parts are trained separately on two fully connected layer models using the same training set and test set. Am I understanding this correctly? What is the purpose of this? Does it have any special meaning?

Thanks!

**Comment by yyou1996** [comment]
Hi @jiaruHithub,

I do not fully understand your question. Would you be able to elaborate more on it? The setting of unsupervised_tu follows https://github.com/sunfanyunn/InfoGraph/tree/master/unsupervised without changes.
